src/main/WorkflowSelection.tsx

```diff
@@ -49,13 +49,12 @@ const WorkflowSelection : React.FC<{}> = () => {
         workflows[selectedWorkflowIndex].description
       );
     } else {
-      return (
-        "And this is where I would put a workflow description.... if I had one!"
-      );
+      return (' ');
```

Comment:
Would it be better to show nothing or _"No workflow description available"_.

Reply:
I don't think “No workflow description available” is helpful for users.
They cannot do anything about it and it doesn't help them to know what the workflow is about.
It's just distracting from other elements.